airflow/utils/session.py

```diff
@@ -0,0 +1,63 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
What do you think to change the name to `db_session.py`? The very word "session" says too little

Reply:
I see your point but `db_session` makes me to wonder what is the other not-db-session?

Reply:
In my opinion, this is not a problem because it will be in utils package.  It often happens that additional utility files are created, if needed.

Reply:
I like session. We already have 'provide_session' decorator rather than 'provide_db_session' and we are fine with it